tableauserverclient/server/endpoint/views_endpoint.py

```diff
@@ -24,9 +24,12 @@ def baseurl(self):
         return "{0}/views".format(self.siteurl)
 
     @api(version="2.2")
-    def get(self, req_options=None):
+    def get(self, req_options=None, usage=False):
         logger.info('Querying all views on site')
-        server_response = self.get_request(self.baseurl, req_options)
+        url = self.baseurl
+        if usage:
+            url += "?includeUsageStatistics=true"
```

Comment:
This could lead to the url being `?includeUsageStatistics=true?page=2` if you pass in request options.  Can you add a test to check that the url being generated in this case would be `?includeUsageStatistics=true&page=2` ?

You'll probably want to amend request options so that it inspects the url for the existence of `?` and then uses an `&` to append options to the query string.